mobile/lib/constants/config.dart

```diff
@@ -10,6 +10,9 @@ class Config {
   static String get airqoApiToken => dotenv.env['AIRQO_API_TOKEN'] ?? '';
   static String get searchApiKey => dotenv.env['SEARCH_API_KEY'] ?? '';
 
+  static double get minimumTextScaleFactor => 1.0;
+  static double get maximumTextScaleFactor => 1.1;
```

Comment:
Overriding minimum and maximum scale factors when user changes the phone font size